draft-ietf-masque-connect-ip-dns.md

```diff
@@ -362,18 +437,19 @@ corresponding ROUTE_ADVERTISEMENT capsule.
 
 # IANA Considerations {#iana}
 
-This document, if approved, will request IANA add the following value to the
+This document, if approved, will request IANA add the following values to the
 "HTTP Capsule Types" registry maintained at
 <[](https://www.iana.org/assignments/masque)>.
 
-Value:
-
-: 0x1ACE79EC (if this document is approved, this value will be
-replaced by a smaller one before publication)
+| Value | Capsule Type |
+| --- | --- |
+| 0x274C0FBB (if this document is approved, this value will be
```

Comment:
Since this PR doesn't change the DNS capsule at all, should we keep the previous value of 0x1ACE79EC?